.github/workflows/dart.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     
     container:
-      image:  google/dart:latest
+      image: cirrusci/flutter:latest
```

Comment:
Sorry for late review.

Why not `image: google/dart:latest` ?

Reply:
Because it has old version of dart. Last updated 8 months ago. I just checked a couple of packages on pub.dev and found this image. You can check it and change as well.

Reply:
Alright making a PR for it.

Reply:
I've merge a PR to resolve CI/CD to run.

Merge Conflicts Resolved too ✅.